### PR TITLE
Implemented json response

### PR DESCRIPTION
**Pull request by Tivgres**

### CODE REVIEW

File: app/controllers/employees_controller.rb [hunk 1]
```diff
@@ -13,9 +13,10 @@ def show
     @employee.build_image unless @employee.image
     @project = @employee.projects.new
     @project.build_image
-    
+
```
Comment (drkmen) [inline app/controllers/employees_controller.rb]: ?

File: spec/controllers/employees_controller_spec.rb [hunk 2]
```diff
@@ -122,18 +122,27 @@
...
       it 'returns a success response' do
         put :update,
-          params: {
-            id: Employee.last.id,
+            params: {
+              id: Employee.last.id,
               employee: {
                 first_name: 'qweqwe',
-                last_name: 'qweqwe'
+                  last_name: 'qweqwe'
```
Comment (drkmen) [inline spec/controllers/employees_controller_spec.rb]: ?

File: spec/controllers/employees_controller_spec.rb [hunk 3]
```diff
@@ -160,16 +169,16 @@
...
-                experience: 'better'
-              }
-            }
+                params: {
+                  employee_id: employee_2.id,
+                  skill_experience: {
+                    id: ResourceSkill.last.id,
+                    experience: 'better'
+                  }
+                }
```
Comment (drkmen) [inline spec/controllers/employees_controller_spec.rb]: ?

File: app/controllers/employees_controller.rb (hunk 1 above)
Comment (Tivgres) [reply app/controllers/employees_controller.rb]: Rubocop style. Removed unused.

File: spec/controllers/employees_controller_spec.rb (hunk 2 above)
Comment (Tivgres) [reply spec/controllers/employees_controller_spec.rb]: Fixed via rubocop style.

File: spec/controllers/employees_controller_spec.rb (hunk 3 above)
Comment (Tivgres) [reply spec/controllers/employees_controller_spec.rb]: Fixed via rubocop style.

**Pull request merged**